wicketd/src/update_tracker.rs

```diff
@@ -151,19 +151,20 @@ impl UpdateTracker {
 
                 match upload_trampoline_phase_2_to_mgs.as_mut() {
                     Some(prev) => {
-                        // We've previously started an upload - does it match this
-                        // update's artifact ID? If not, cancel the old task (which
-                        // might still be trying to upload) and start a new one with
-                        // our current image.
+                        // We've previously started an upload - does it match
+                        // this update's artifact ID? If not, cancel the old
+                        // task (which might still be trying to upload) and
+                        // start a new one with our current image.
                         //
-                        // TODO-correctness If we still have updates running that
-                        // expect the old image, they're probably going to fail.
-                        // Should we handle that more cleanly or just let them fail?
+                        // TODO-correctness If we still have updates running
+                        // that expect the old image, they're probably going to
+                        // fail. Should we handle that more cleanly or just let
+                        // them fail?
```

Comment:
Now that we've made it so that a new repository can't be uploaded while updates are running, I think we can just make old running updates (which definitely can't be spun up by this copy of wicket) fail.

Reply:
Ah good point. This was just a formatting change, but I'll remove this TODO entirely. Nice catch.